database/sampleDataQueries.sql

```diff
@@ -0,0 +1,18 @@
+SET FOREIGN_KEY_CHECKS = 0;
```

Comment:
Thanks for adding this file! It's a good idea to have some sample data in the DB